resources/queries/targetedms/QCMetricEnabled_transitionArea.sql

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016-2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-- Approximate the checks in GeneralTransition.isQuantitative()
+SELECT Id, FragmentType, Quantitative FROM Transition t
+    WHERE
+        (Quantitative = TRUE) OR (Quantitative IS NULL AND
+                                  (FragmentType != 'precursor' AND
+                                  t.GeneralPrecursorId.GeneralMoleculeId.PeptideGroupId.RunId IN (
```

Comment:
is this indentation intentional?

Reply:
Kind of. IntelliJ has strange ideas for how to indent this SQL and I got tired of fighting it.